dts/arm/silabs/siwg917.dtsi

```diff
@@ -19,9 +19,14 @@
 		};
 	};
 
-	sram0: memory@0 {
+	sram0: memory@400 {
 		compatible = "mmio-sram";
-		reg = <0x00000000 DT_SIZE_K(191)>;
+		reg = <0x400 DT_SIZE_K(191)>;
+	};
```

Comment:
Could you place this change in a separate commit and explain why you think it is required?

Reply:
I reverted this.
SRAM starts from 0x400 in the linker scripts of the WiseConnect.
And only if it stars from 0x400 then the 191kB size is true (otherwise it should be 192kB).